### PR TITLE
(PE-33901) Add dumpling to codeowners

### PR DESCRIPTION
**Pull request by KNBAnderson**

### CODE REVIEW

**Comment by KNBAnderson** [comment]
I wasn't sure if this should be on 6x or on main

**Comment by jonathannewman** [comment]
main should be fine.

File: CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
-* @puppetlabs/puppetdb
-/docker/ @puppetlabs/pupperware
+* @puppetlabs/dumpling
+/docker/ @puppetlabs/dumpling
```
Comment (jonathannewman) [inline CODEOWNERS:2]: We can eliminate this specification as line 1 covers the whole repo.

**Review by jonathannewman — CHANGES_REQUESTED** [review]
The scope of code owners haas duplication.  It can be more simple.

**Pull request merged**